docs/content/middlewares/forwardauth.md

```diff
@@ -266,7 +266,12 @@ http:
 
 #### `tls.caOptional`
 
-TODO add description.
+The policy followed by the server for TLS Client Authentication.
```

Comment:
```suggestion
Policy used for the secured connection with TLS Client Authentication to the authentication server.
```

Reply:
- Stay coherent with the other sentences (removing the `The` at the beginning)
- Clarify what "server" is